package-lock.json

```diff
@@ -311,7 +311,7 @@
     },
     "assertion-error": {
       "version": "1.0.2",
-      "resolved": "https://registry.npmjs.org/assertion-error/-/assertion-error-1.0.2.tgz",
+      "resolved": "https://sf-npm.mediarithmics.com/repository/npm-mics/assertion-error/-/assertion-error-1.0.2.tgz",
```

Comment:
This is not the default registry.

Reply:
Good catch. I think something got mixed up in my local setup. Fixing it now.